_pt-br/tour/mixin-class-composition.md

```diff
@@ -8,7 +8,7 @@ partof: scala-tour
 
 num: 5
 next-page: higher-order-functions
-previous-page: traits
+previous-page: tuples
 language: pt-br
 ---
 _Nota de tradução: A palavra `mixin` pode ser traduzida como mescla, porém é preferível utilizar a notação original_
```

Comment:
This should be replaced by the stub to indicate it hasn't been translated yet.

Reply:
There _is_ an explanation, but it does not correspond to the English version of the page.

Reply:
Ah sorry, I got confused by the fact that this page is actually about mixins, and it's just the "next" pointer that's being updated.